lms/static/scripts/frontend_apps/components/test/ErrorDisplay-test.js

```diff
@@ -78,6 +78,28 @@ describe('ErrorDisplay', () => {
     );
   });
 
+  it('omits support link when supportLink is false', () => {
+    fakeFormatErrorDetails.returns('foo: bar');
+    fakeFormatErrorMessage.returns('Failed to fetch files: Canvas says no');
+    const error = new Error('');
+    // Build an ErrorLike object that resembles an APIError
+    error.errorCode = 'some_backend_error_code';
+
+    const wrapper = mount(
+      <ErrorDisplay
+        description="Failed to fetch files"
+        error={error}
+        supportLink={false}
+      />,
+    );
+
+    const link = wrapper
+      .find('Link')
+      .filterWhere(n => n.text() === 'open a support ticket');
+
+    assert.isFalse(link.exists());
```

Comment:
Since the element wrapping the support link has a `data-testid` attribute, you can also do something like this:

```suggestion
    assert.isFalse(wrapper.exists('[data-testid="error-links"]'));
```